tests/functional/test_config.py

```diff
@@ -54,6 +54,71 @@ def test_ethereum_network_configs(config, temp_config):
         assert actual.rinkeby.block_time == 15
 
 
+def test_network_gas_limit_default(config):
+    eth_config = config.get_config("ethereum")
+
+    assert eth_config.rinkeby.gas_limit == "auto"
+    assert eth_config.local.gas_limit == "auto"
+
+
+@pytest.mark.parametrize("gas_limit", ("auto", "max"))
+def test_network_gas_limit_string_config(gas_limit, config, temp_config):
+    eth_config = {
```

Comment:
potentially could make a helper method or fixture to share this `eth_config` structure

Reply:
Updated in e002111f and also fixed the documentation of the bad hex string test